backend/cfg/vectorize.ml

```diff
@@ -134,7 +138,327 @@ module Dependency_graph = struct
       ~terminator:(fun instruction -> print_node (`Terminator instruction))
 end
 
-let dump ppf cfg_with_layout ~(dependency_graph : Dependency_graph.t) ~msg =
+module Adjacent_memory_accesses = struct
```

Comment:
Add an interface and a comment documenting the assumptions under which two loads for example are considered adjacent.